catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/metrics/BlackHoleMetricsStore.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright 2023 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics;
```

Comment:
Why do you put metrics package under "web"?

Reply:
metrics package is used only by Iceberg REST service,  and most Iceberg REST services-related files are located in `web`, this is not a good name, maybe rename it to `service`?

Reply:
done